cfme/tests/cloud/test_providers.py

```diff
@@ -123,6 +126,7 @@ def test_provider_add_with_bad_credentials(provider):
 @pytest.mark.smoke
 @pytest.mark.usefixtures('has_no_cloud_providers')
 @test_requirements.discovery
+@pytest.mark.meta(blockers=[BZ(1517061, forced_streams=["5.9"])])
```

Comment:
REQUIRED: this blocker applies only to EC2. and RHOS/Azure/GCE should not be uncollected